### PR TITLE
Feature/brandissue

### PR DESCRIPTION
**Pull request by navin9584**

### CODE REVIEW

File: v1.0/backend/reports/Report_Hindi_1634793577959.json [hunk 1]
```diff
@@ -0,0 +1 @@
...nId":"ROLLNUMBER3_Q12","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q13","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q14 ","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q15","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q16","obtainedMarks":"1"},{"questionId":"ROLLNUMBER3_Q17","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q18","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q19","obtainedMarks":"1"},{"questionId":"ROLLNUMBER3_Q20","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q21","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q22","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q23","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q24","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q25","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q26","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q27","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q28","obtainedMarks":"0"},{"questionId":"ROLLNUMBER3_Q29","obtainedMarks":"1"},{"questionId":"ROLLNUMBER3_Q30","obtainedMarks":"0"}],"createdOn":"1634793516262","studentId":"1380802","section":"D","securedMarks":7,"totalMarks":0,"schoolId":"1","examDate":"24/09/2021","subject":"Hindi","classId":"3"}],"totalPages":1,"currentPage":1}
```
Comment (dileep-gadiraju) [inline v1.0/backend/reports/Report_Hindi_1634793577959.json]: what is this file for?

File: v1.0/frontend/SaralApp/src/configs/config.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 const configs = {
     // BASE_URL: 'http://uppat.idc.tarento.com:3000'
     // BASE_URL: 'http://44.241.36.235:3000'
+<<<<<<< HEAD
+    BASE_URL: 'http://192.168.253.164:3000'
```
Comment (dileep-gadiraju) [inline v1.0/frontend/SaralApp/src/configs/config.js]: resolve merge conflicts here

**Comment by apurvabisht97** [comment]
This file generates when we save our marks.

File: v1.0/frontend/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (dileep-gadiraju) [inline v1.0/frontend/package-lock.json]: this file is unwated right? this is expected only in v1.0/frontend/SaralApp folder

File: v1.0/frontend/package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (dileep-gadiraju) [inline v1.0/frontend/package.json]: Is this file needed in v1.0/frontend folder?

File: v1.0/backend/reports/Report_Hindi_1634793577959.json (hunk 1 above)
Comment (dileep-gadiraju) [reply v1.0/backend/reports/Report_Hindi_1634793577959.json]: remove this file

File: v1.0/frontend/SaralApp/src/modules/myScanScreens/MyScanComponent.js [hunk 5]
```diff
@@ -190,7 +186,6 @@ class MyScanComponent extends Component {
             } else {
             }
         } catch (err) {
-            console.warn(err);
```
Comment (dileep-gadiraju) [inline v1.0/frontend/SaralApp/src/modules/myScanScreens/MyScanComponent.js:193]: keep this.

**Review by dileep-gadiraju — APPROVED** [review]
